src/uas/UAS.cc

```diff
@@ -147,22 +147,20 @@ UAS::UAS(MAVLinkProtocol* protocol, Vehicle* vehicle) : UASInterface(),
     blockHomePositionChanges(false),
     receivedMode(false),
 
-    // Initialize HIL sensor noise variances to 0.  If user wants corrupted sensor values they will need to set them
-    // Note variances calculated from flight case from this log: http://dash.oznet.ch/view/MRjW8NUNYQSuSZkbn8dEjY
-    // TODO: calibrate stand-still pixhawk variances
-    xacc_var(1.2914f),
-    yacc_var(0.7048f),
-    zacc_var(1.9577f),
-    rollspeed_var(0.8126f),
-    pitchspeed_var(0.6145f),
-    yawspeed_var(0.5852f),
-    xmag_var(0.4786f),
-    ymag_var(0.4566f),
-    zmag_var(0.3333f),
-    abs_pressure_var(1.1604f),
-    diff_pressure_var(1.1604f),
-    pressure_alt_var(1.1604f),
-    temperature_var(1.4290f),
+    // measured pixhawk sensor noise variances

```

Comment:
If you want to simulate vibration, then the levels must be airframe specific, not global defaults. At the least, they should be modeled based on the number, location and type of motors on the vehicle. Are there any other sources of vibration which this was supposed to model?